components/ui/LeadsList/LeadsList.tsx

```diff
@@ -0,0 +1,121 @@
+import { LEADS_DATA } from '@/utils/constants/leadslist-constant';
+import Image from 'next/image';
+import plus from '/public/leadslist-icons/add-circle.png';
+import phone from '/public/leadslist-icons/call.png';
+import clock from '/public/leadslist-icons/clock.png';
+import more from '/public/leadslist-icons/more_vert.png';
+
+// Define a function to get the appropriate color based on status
+const getStatusColor = (status: string) => {
```

Comment:
I will suggest you use this method as a utility function, so you have to move it to `utils -> helpers -> common-helpers.ts` file.